artem/JsBeginingPractice/ArrayAndObjects/script.js

```diff
@@ -0,0 +1,843 @@
+function addTwoToEveryNumberInTheList (list) {
+  // an example for you to see how the tests work
+  
+  const newList = [];
+
+  for (let i = 0; i < list.length; i++) {
+    newList.push(list[i] + 2);
+  }
+  
+  return newList;
+} 
+
+function findLargestNumber(list) {
+  // return the largest number in array
+  
+  return Math.max.apply(null, list);
+}
+
+function makeDogObject() {
+  
+    const dog = {
+    name: "Fido",
```

Comment:
indentation is struggling here
example:
```
const dog = {
    name: 'Fido',
    ....
}

```

Reply:
got it.